src/actions/__tests__/range-test.js

```diff
@@ -3,7 +3,7 @@ import getRange from '../range'
 describe('actions/getRange', () => {
   it('returns range between 2 and totalPage if totalPage' +
      'is larger than preset range', () => {
-    expect(getRange(1, 2, 3)).toEqual([2])
+    expect(getRange(1, 1, 3)).toEqual([2])
```

Comment:
And why does it return 2 when the values is `1, 1, 3`?
Use a named constant to describe what `2` is in both the test and the code will be useful.